tests/NuGet.Services.AzureSearch.FunctionalTests/Config/Dev.json

```diff
@@ -0,0 +1,18 @@
+﻿{
```

Comment:
Looks like I messed up merging @shishirx34's changes into mine: https://github.com/NuGet/NuGet.Services.Metadata/commit/140c7cb225abeaa40f26fcb7191b3a4973619fd2#diff-631c1158b1b69ee2e83ea63d5f7483ce

I will open up a pull request and delete this file 😢

Reply:
https://github.com/NuGet/NuGet.Services.Metadata/pull/576